ta/pkcs11/src/object.c

```diff
@@ -369,6 +369,10 @@ enum pkcs11_rc entry_create_object(struct pkcs11_client *client,
 	if (rc)
 		goto out;
 
+	rc = check_attr_checksum(head);
```

Comment:
I would prefer this to be called before `check_created_attrs_against_processing()`, for consistency:
first set all attributes, then check they comply, then create the object.